cea/datamanagement/databases_verification.py

```diff
@@ -0,0 +1,83 @@
+from __future__ import division
+from __future__ import print_function
+
+
+COLUMNS_ZONE_GEOMETRY = ['Name', 'floors_bg', 'floors_ag', 'height_bg', 'height_ag']
+COLUMNS_DISTRICT_GEOMETRY = ['Name', 'height_ag', 'floors_ag', 'floors_bg']
+COLUMNS_ZONE_AGE = ['built', 'roof', 'windows', 'partitions', 'basement', 'HVAC', 'envelope']
+COLUMNS_ZONE_OCCUPANCY = ['MULTI_RES', 'OFFICE', 'RETAIL', 'SCHOOL', 'HOTEL', 'GYM', 'HOSPITAL', 'INDUSTRIAL',
+                          'RESTAURANT','SINGLE_RES', 'SERVERROOM', 'SWIMMING', 'FOODSTORE', 'LIBRARY', 'COOLROOM', 'PARKING']
+
+
+def assert_columns_names(zone_df, columns):
+
+    try:
+        zone_test = zone_df[columns]
+    except ValueError:
+        print("one or more columns in the Zone or District input files is not compatible with CEA, please ensure the column" +
+              " names comply with:", columns)
+
+def assert_input_geometry_acceptable_values_floor_height(zone_df):
+
+    # Rule 0. nothing can be negative
+    rule0 = zone_df.where(zone_df < 0).any().any()
+    if rule0:
+        raise Exception('There are negative values in your geometry. This is not possible to simulate in CEA at the moment'
+                        ' Please verify your Zone or District shapefile file')
+
+    # Rule 1. Floors above ground cannot be less than 1 or negative.
+    rule1_1 = zone_df['floors_ag'].where(zone_df['floors_ag'] < 1).any()
```

Comment:
this can also be written as `(zone_df.floors_ag >= 1).all()` with the opposite truth value (being `True` if the rule is met) - i'm not quite sure what is easier to understand: whether a rule is satisfied or, as in the code, whether it is broken. intuitively, i think this PR does it backwards, but with the commenting of the code it's really clear what is meant. no need to change for now - as mentioned above, a more declarative approach, as in specifying constraints on columns in a declarative file and having a rule checker go through them all, could make this clearer...